src/server.ts

```diff
@@ -108,6 +109,7 @@ async function search(
 		message: `Received search request: ${criteriaStr}`,
 	});
 
+	await refreshIndexerCaps();
```

Comment:
this feels a bit heavy… let's keep an eye on it. Caps basically never change so I think we could probably get away with something like a weekly or even less frequent job to update caps

Reply:
I was thinking the same thing, given they don't update or change.

Reply:
Prowlarr checks daily for indexer updates. I think that would be a good rate if doing it by job.

Reply:
Agreed